e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+describe('Integrations', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testChannelUrl1;
+    let testChannelUrl2;
+    let commandURL;
+
+    const commandTrigger = 'test-ephemeral';
+    const timestamp = Date.now();
+
+    before(() => {
```

Comment:
Please add `cy.requireWebhookServer();` so this will fail immediately when such server is not running.